src/Testcontainers/Configurations/Modules/Databases/CosmosDbTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,67 @@
+namespace DotNet.Testcontainers.Configurations
+{
+    using DotNet.Testcontainers.Builders;
+    using JetBrains.Annotations;
+
+    [PublicAPI]
+    public class CosmosDbTestcontainerConfiguration : TestcontainerDatabaseConfiguration
+    {
+        private const string CosmosDbImage = 
+            "mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator";
+
+        private const int CosmosDbPort = 8081;
+
+        private const string CosmosDbDefaultKey =
+            "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
```

Comment:
What kind of secret is that?

Reply:
It's the default key for the cosmosDB emulator, it's in the docs so not leaking anything here.

Reply:
https://docs.microsoft.com/de-de/azure/cosmos-db/local-emulator?tabs=ssl-netstd21#sql-api